section2/exercises/if-statements.js

```diff
@@ -62,11 +68,15 @@ When numQuarters = 2, program should log "I have enough money for a gumball"
 When numQuarters = 3, program should log "I have enough money for a gumball"
 */
 
+//var numQuarters = 3;
+
 var numQuarters = 0;
 
-// Write your conditional here
-console.log("I have enough money for a gumball");
-console.log("I don't have enough money for a gumball");
+if (numQuarters >= 2) {
+  console.log("I have enough money for a gumball.");
+} else { (numQuarters < 2)
```

Comment:
Careful of your syntax here. We don't include a condition with `else`.

And technically this condition, `(numQuarter < 2)`, is recognized as valid JS code but since it comes after the open curly bracket, it executes before the console.log (but doesn't do anything).